opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp

```diff
@@ -546,7 +547,7 @@ class Well {
       keyword.
     */
     std::map<int, std::vector<Connection>> getCompletions() const;
-
+    bool hasCompletion(int completion) const;
```

Comment:
Looks ok, but I wonder what that `int completion` is. I think it is an "identifier" and not an index, I guess what I'm really looking for is a way to discover (comment for example) what number I am supposed to plug in here.

Reply:
Comment added.